scripts/joinmarket-qt.py

```diff
@@ -1464,71 +1464,84 @@ def openQRCodePopup(self, address):
         popup.show()
 
     def updateWalletInfo(self, walletinfo=None):
-        nm = jm_single().config.getint("GUI", "max_mix_depth")
-        l = self.walletTree
+        max_mix_depth_count = jm_single().config.getint("GUI", "max_mix_depth")
 
+        previous_expand_states = []
         # before deleting, note whether items were expanded
-        esrs = []
-        for i in range(l.topLevelItemCount()):
-            tli = l.invisibleRootItem().child(i)
+        for i in range(self.walletTree.topLevelItemCount()):
+            tli = self.walletTree.invisibleRootItem().child(i)
             # must check top and also the two subitems (branches):
             expandedness = tuple(
                 x.isExpanded() for x in [tli, tli.child(0), tli.child(1)])
-            esrs.append(expandedness)
-        l.clear()
-        if walletinfo:
-            rows, mbalances, xpubs, total_bal = walletinfo
-            if jm_single().config.get("BLOCKCHAIN", "blockchain_source") == "regtest":
-                self.wallet_name = mainWindow.testwalletname
+            previous_expand_states.append(expandedness)
+
+        self.walletTree.clear()
+
+        # Skip the remaining of this method if wallet info doesn't exist
+        if walletinfo == None:
+            return
+            
+        rows, mbalances, xpubs, total_bal = walletinfo
+        if jm_single().config.get("BLOCKCHAIN", "blockchain_source") == "regtest":
+            self.wallet_name = mainWindow.testwalletname
+        else:
+            self.wallet_name = os.path.basename(
+                mainWindow.wallet_service.get_storage_location())
+        if total_bal is None:
+            if jm_single().bc_interface is not None:
+                total_bal = " (syncing..)"
             else:
-                self.wallet_name = os.path.basename(
-                    mainWindow.wallet_service.get_storage_location())
-            if total_bal is None:
-                if jm_single().bc_interface is not None:
-                    total_bal = " (syncing..)"
-                else:
-                    total_bal = " (unknown, no blockchain source available)"
-            self.label1.setText("CURRENT WALLET: " + self.wallet_name +
-                                ', total balance: ' + total_bal)
-            l.show()
+                total_bal = " (unknown, no blockchain source available)"
+        self.label1.setText("CURRENT WALLET: " + self.wallet_name +
+                            ', total balance: ' + total_bal)
+        self.walletTree.show()
 
         if jm_single().bc_interface is None and self.wallet_name != 'NONE':
             return
 
-        for i in range(nm):
+        for mix_depth in range(max_mix_depth_count):
```

Comment:
nit - maybe use `mixdepth` instead of `mix_depth` for consistency of other parts of the same file.

Reply:
Done :)